Gruntfile.cjs

```diff
@@ -10,34 +12,39 @@ module.exports = function(grunt) {
 
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
-    browserify: {
-      dist: {
-        src: ['./src/RosLibBrowser.js'],
-        dest: './build/roslib.js'
-      }
-    },
-    jshint: {
-      options: {
-        jshintrc: true
+    shell: {
+      build: {
+        command: 'rollup -c'
+      },
+      lint: {
+        command: 'eslint Gruntfile.cjs ./src/*.js ./src/**/*.js ./test/*.js'
+      },
+      'lint-fix': {
+        command: 'eslint --fix Gruntfile.cjs ./src/*.js ./src/**/*.js ./test/*.js'
+      },
+      mochaTest: {
+        command: 'mocha ./test/*.test.js'
+      },
+      mochaExamples: {
+        command: 'mocha ./test/examples/*.js'
+      },
+      mochaTcp: {
+        command: 'mocha ./test/tcp/*.js'
       },
-      files: [
-        './Gruntfile.js',
-        './src/**/*.js'
-      ]
     },
     karma: {
       options: {
         singleRun: true,
-        browsers: process.env.CI ? ['FirefoxHeadless'] : ['Firefox']
+        browsers: process.env.CI ? ['ChromeHeadless'] : ['Chrome']
```

Comment:
I switched to Chrome because I was having issues getting the `importmap` polyfill working in Firefox:

- https://github.com/guybedford/es-module-shims/issues/186
- https://github.com/guybedford/es-module-shims/issues/187

I decided to just get this all working natively first, then we can polyfill `importmap` in Firefox later if we still want to. But MS Edge is now built on Chromium, so at least the tests currently cover a large user base.